spec/features/shelters/cannot_delete_with_approved_pets_spec.rb

```diff
@@ -68,4 +71,21 @@
 
       expect(page).to_not have_content(@pet_2.name)
   end
+
+
+  it "deletes the reviews for the shelter when the shelter is deleted" do
+    reviews_count = Review.count
```

Comment:
Make sure that both reviews in the setup were created.

Reply:
I like how you managed to test that